src/command.rs

```diff
@@ -148,7 +154,7 @@ impl<'a> Commands<'a> {
                     bail!("Duplicate parameter names: {}", &duplicates.join(", "));
                 }
             }
-            let command_list = command_strings.collect::<Vec<&str>>();
+            let command_list = command_strings;
```

Comment:
Hm. Do we need both? Or could we use `command_list` above?

Reply:
I removed the unneeded command_list variables